### PR TITLE
Skip TestTriggerNoFinalizerOnBrokerNotFound test

### PR DESCRIPTION
**Pull request by pierDipi**
Skip TestTriggerNoFinalizerOnBrokerNotFound test since it's flaky

### CODE REVIEW

**Comment by pierDipi** [comment]
Worked
/test pull-knative-sandbox-eventing-kafka-broker-integration-tests

**Comment by pierDipi** [comment]
2/2 Worked
/test pull-knative-sandbox-eventing-kafka-broker-integration-tests

**Comment by pierDipi** [comment]
4/4 Worked
/test pull-knative-sandbox-eventing-kafka-broker-integration-tests

**Comment by pierDipi** [comment]
5/5 Worked
/test pull-knative-sandbox-eventing-kafka-broker-integration-tests

**Comment by pierDipi** [comment]
6/6 Worked
/test pull-knative-sandbox-eventing-kafka-broker-integration-tests

**Comment by pierDipi** [comment]
7/7 Worked
/test pull-knative-sandbox-eventing-kafka-broker-integration-tests

**Comment by pierDipi** [comment]
8/8 Worked
/test pull-knative-sandbox-eventing-kafka-broker-integration-tests

**Comment by slinkydeveloper** [comment]
Is this the test that is failing in other PRs as well? Can we merge it?

**Comment by pierDipi** [comment]
9/9 Worked
/test pull-knative-sandbox-eventing-kafka-broker-integration-tests

**Comment by slinkydeveloper** [comment]
There are some glitches in the verify deps GH action, i'm going to check it out

**Review by slinkydeveloper — APPROVED** [review]
/lgtm

As soon as this works for you, merge it, it's blocking the release. If you don't reply for 17:00 CEST, I remove the hold :)

**Comment by pierDipi** [comment]
No there are disabled tests.

File: test/e2e-tests.sh
Comment (slinkydeveloper) [inline test/e2e-tests.sh:46]: If you remove the comments the test fails?

**Review by slinkydeveloper — CHANGES_REQUESTED** [review]

File: test/e2e-tests.sh
Comment (slinkydeveloper) [reply test/e2e-tests.sh:46]: If that's ok for you, can we disable the flaky test or revert this patch https://github.com/knative-sandbox/eventing-kafka-broker/commit/63c8083b249a4be5f342a27555482bbe6fd8f7df and then tackle it after the release? I can take care of fixing the issue.

Comment (slinkydeveloper) [reply test/e2e-tests.sh:46]: I've just checked again the fix and it seems correct to me, so maybe it's just the test?

Comment (pierDipi) [reply test/e2e-tests.sh:46]: The test is not in these suites, it's in `e2e_new`, I disabled those for getting a failure.

Comment (slinkydeveloper) [reply test/e2e-tests.sh:46]: Can you do it in this PR? Otherwise i'll do it in another one

**Review by slinkydeveloper — APPROVED** [review]
/lgtm

**Pull request merged**
